File: Atividades Udemy/Java/secao06/atividade09.java
package secao06;

import java.util.Scanner;

public class atividade09 {

	public static void main(String[] args) {
		float indice;
		Scanner teclado = new Scanner(System.in);
		
		System.out.println("Informe o índice de poluição: ");
		indice = teclado.nextFloat();
		
		if(indice >= 0.3 && indice < 0.4) {
			System.out.println("Atenção: Indústrias do 1º grupo devem suspender as atividades.");
		}else if(indice >= 0.4 && indice < 0.5) {
			System.out.println("Atenção: Indústrias do 1º e 2º grupo devem suspender as atividades.");
		}else if (indice >= 0.5) {
			System.out.println("Alerta: As indústrias de todos os grupos devem suspender as atividades.");
		}else {
			System.out.println("Índice de poluição normalizado.");
		}
		teclado.close();

	}

}

File: Atividades Udemy/Python/secao06/atividade09.py
indice = float(input("Informe o índice de poluição: "))

if(indice >= 0.3 and indice < 0.4):
    print("As indústrias do 1º grupo devem suspender as atividades.")
elif(indice >= 0.4 and indice < 0.5):
    print("As indústrias do 1º e 2º grupo devem suspender as atividades.")
elif(indice >= 0.5):
    print("ALERTA: As indústrias de todos os grupos devem suspender as atividades.")
else:
    print("Índice de poluição aceitável.")